rsopt/configuration/jobs.py

```diff
@@ -123,11 +123,16 @@ def sim_dirs_required(self) -> bool:
             return True
         return False
 
-    @property
-    def ignored_files(self) -> list:
+    def ignored_files(self, with_path: bool = False) -> list:
         # Get files that should be ignore by sirepo.lib parse
```

Comment:
Not the proper docstring format

Reply:
Added in 2df2a8a027aa54ab637330a7d0cde9c14861fbe2